schema.graphql

```diff
@@ -70,6 +70,10 @@ type Account @entity {
   receivedTransfers: [Transfer!]! @derivedFrom(field: "to")
   "Outgoing transfers"
   sentTransfers: [Transfer!]! @derivedFrom(field: "from")
+  "Staked token amount"
+  stakedTokens: BigInt!
```

Comment:
should this conform to `stakedTokensRaw` + `stakedTokens`?
Applies to almost all of these fields? I was of the mind of just letting conversion be done by the consuming application.